README.md

```diff
@@ -140,7 +140,7 @@ PHP: SDK|Every Wednesday at 10:30 PT|[Google Doc](https://docs.google.com/docume
 Python: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)|[Slack](https://cloud-native.slack.com/archives/C01PD4HUVBL)|
 Ruby: SDK|Every Tuesday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)|[Slack](https://cloud-native.slack.com/archives/C01NWKKMKMY) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions)|
 Rust: SDK|Every other week on Tuesday alternating between 07:00 and 15:00 PT|[Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)|[Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
-Swift: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1ShEFMywIV4LJcDYCNy41zkq8RR1sMq-tSvMBHngHcuk/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C01NCHR19SB)||
+Swift: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/17KAyOXt-0Ti3PVB5_tLJi3lFeFNfp_98Ji3NocBqLQI/edit)|[Slack](https://cloud-native.slack.com/archives/C01NCHR19SB)||
```

Comment:
This document is not publicly accessible. Please make sure it's world-readable (maybe allow comment access, most SIGs even have them publicly editable although that's probably not ideal).

Please also make sure it's owned by the OTel GC account - see https://github.com/open-telemetry/community/issues/1147.

Reply:
@nachoBonafonte any update? The document is still private.

Reply:
ping @open-telemetry/swift-maintainers